### PR TITLE
Location Attributes CRUD w/ repository pattern

### PR DESCRIPTION
**Pull request by SilviuRusu3**

### CODE REVIEW

File: HomeFinder/Models/ListAttributesRepository.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace HomeFinder.Models
+{
+    public class ListAttributesRepository //: IAttributesRepository
```
Comment (andreisamoila74) [inline HomeFinder/Models/ListAttributesRepository.cs]: if we are not using this class anywhere, we should delete it

File: HomeFinder/Models/ListAttributesRepository.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        //I kept it because it's work that I have done and don't want to be lost
+        //Because of Repository pattern it doesn't interfere with the functionality of the program
+        //as it is no longer being used
+        //It doesn't take fully in consideration the changes in Rank of the attributes
+        List<LocationAttributes> _attributesList;
+        public ListAttributesRepository()
+        {
+            _attributesList = new List<LocationAttributes>
+           {
+               new LocationAttributes{Id = 1, Name = "Aproape de parc", Rank = 1},
```
Comment (andreisamoila74) [inline HomeFinder/Models/ListAttributesRepository.cs]: don't hardcode values if you don't need them

**Review by andreisamoila74 — CHANGES_REQUESTED** [review]
- also, please a short loom video with a briefly explanation of this pull request
- you should squash the commits, because there 2 are commits with the same name

File: HomeFinder/Models/ListAttributesRepository.cs (hunk 1 above)
Comment (SilviuRusu3) [reply HomeFinder/Models/ListAttributesRepository.cs]: I will delete it

File: HomeFinder/Models/ListAttributesRepository.cs (hunk 2 above)
Comment (SilviuRusu3) [reply HomeFinder/Models/ListAttributesRepository.cs]: This was just for learning. In the database class, I didn't hardcode anything.

**Review by andreisamoila74 — APPROVED** [review]

**Pull request merged**